### PR TITLE
TCM-424 | Improve `Player` screen

### PR DESCRIPTION
**Pull request by VivekV95**
## Description
Make visual improvements to the `Player` screen

### Summary

- [x] Improve dynamic margins
- [x] Introduce close button in landscape mode to go back to portrait
- [x] Remove scrubber
- [x] Fix cell margins in `Discover` 
### Issue

[TCM-424](https://hyfnla.atlassian.net/browse/TCM-424)

### Video/Screenshot

[Player](https://drive.google.com/file/d/1sVkfVaxzBEufflpqljCjYB7m0UgBLB3f/view?usp=sharing)
[Discover](https://drive.google.com/file/d/1o8kBnFFPkcXiESXXFHGMXLFO7ZqBCdhB/view?usp=sharing)
[Discover empty](https://drive.google.com/file/d/1Pt6mgUGuuqcFZsOBCroLPQFfzlgECTLt/view?usp=sharing)

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
     <color name="transparent">#00000000</color>
     <color name="black">#000000</color>
+    <color name="blackOther">#111113</color>
```
[the remark this answers is not in the document]
Comment (onodude) [reply app/src/main/res/values/colors.xml:6]: please remove this new color

**Review by onodude — CHANGES_REQUESTED** [review]
added a small comment

File: app/src/main/res/values/colors.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (VivekV95) [reply app/src/main/res/values/colors.xml:6]: That's the color of the background in the figma design, should we just use black instead?

[the remark this answers is not in the document]
Comment (onodude) [reply app/src/main/res/values/colors.xml:6]: yes

[the remark this answers is not in the document]
Comment (VivekV95) [reply app/src/main/res/values/colors.xml]: [Done](https://github.com/hyfn/trx-consumer-android/pull/84/commits/89c30dd9fd05133fa5433891ffbde77b60ebc02a#diff-d21b36fc9e8a3dfabd930b6500fba09e4199283a9c1783b6fe599a9d1433d04fL6)

**Comment by VivekV95** [comment]
Updated demos

**Review by onodude — APPROVED** [review]
thank you!

**Pull request merged**